### DISCUSSION TITLE
Does MQTT in NATS supports features like Webhooks and Mountpoints which are available in Verne MQ?

### DISCUSSION THREAD

**Discussion by BenBasilTomy** [nats-io/nats-server#5507] [Ideas]
I am planning to replace Verne MQ with NATS. So does MQTT in NATS supports features like Webhooks and Mountpoints which are available in Verne MQ?

**Comment by derekcollison** [comment]
Not a this time. I think I understand webhooks, and with our recently annnounced HTTP gateway there could be something possible there.

What are mountpoints in this context? What are they useful for?

**Reply by BenBasilTomy** [reply]
Normally, an MQTT broker hosts one single topic tree. This means that all topics are accessible to all publishers and subscribers. Mountpoints are a way to host multiple topic trees in a single broker.

**Reply by derekcollison** [reply]
If I am reading this correctly, this matches with our notion of accounts, which are separate addressable subject spaces. They can optionally be connected via export and imports of streams and services as well.

https://docs.nats.io/running-a-nats-service/configuration/securing_nats/accounts
